test/runtests.jl

```diff
@@ -287,6 +287,14 @@ end
     @test X′ == X
     # test for presence of superfluous wrapping
     @test typeof(X′) == typeof(X)
+
+    # Treating vectors as 1-column tables:
+    x = [1,2,3,4,5]
+    tbl = Tables.Columns(Tables.table(x))
+    @test keys(tbl) == [:Column1]
+    x′ = Tables.matrix(tbl)
+    @test x′ == reshape(x, :, 1)
+    @test Tables.matrix(Tables.table(x)) == x
```

Comment:
are you convinced `matrix ∘ table` should be no-op? I think people would assume that `matrix` would guarantee to return a matrix.

Reply:
changed